provider/oracle/images.go

```diff
@@ -20,6 +21,63 @@ var windowsServerMap = map[string]string{
 	"Microsoft_Windows_Server_2008_R2": "win2018r2",
 }
 
+// defaultImages is a list of official Ubuntu images available on Oracle cloud
+// TODO (gsamfira): seed this from simplestreams
+var defaultImages = []string{
+	"Ubuntu.12.04-LTS.amd64.20170417",
+	"Ubuntu.14.04-LTS.amd64.20170405",
+	"Ubuntu.16.04-LTS.amd64.20170221",
+	"Ubuntu.16.10.amd64.20170330",
+}
+
+// ensureImageInventory populates the image inventory for the current user
+// with official Ubuntu images
+func ensureImageInventory(c EnvironAPI) error {
```

Comment:
Even with simplestreams, this function will exist in some form as the images still need to be copied to the users account right? Can you add a TODO to add a test